CHANGELOG.md

```diff
@@ -1,8 +1,29 @@
 ## Solidus Auth Devise v1.5.0 (master, unreleased)
 
-* Add call to set_current_order on sign in. This replaces a before filter that is being eliminated from Solidus controllers where set_current_order was called excessively.
+* Add call to set_current_order on sign in. This replaces a before filter that
+  is being eliminated from Solidus controllers where set_current_order was
+  called excessively.
+
 * Update backend views to only reference backend routes (#57)
 
+* Devise dependency updates
+
+  The locked versions of Devise and Devise-Encryptable have been updated.
+  Devise, in particular, has been bumped to a new major version, which removes
+  support for the following:
+
+  - Rails 3.2. and 4.0
```

Comment:
Are these versions of Rails supported in `solidus_version = [">= 1.0.6", "< 2"]`?